jax/api.py

```diff
@@ -75,7 +76,9 @@
 
 def _check_callable(fun):
   if not callable(fun):
-    raise TypeError("Expected a callable value, got {}".format(fun))
+    raise TypeError(f"Expected a callable value, got {fun}")
```

Comment:
We switched to Python 3 just recently, we should start using f strings more (although technically we have not told users to switch off Python 3.5). Thanks!